src/codeflare_sdk/ray/cluster/build_ray_cluster.py

```diff
@@ -338,7 +340,9 @@ def get_worker_container_spec(
             cluster.config.worker_memory_limits,
             cluster.config.worker_extended_resource_requests,
         ),
-        volume_mounts=VOLUME_MOUNTS,
+        volume_mounts=generate_custom_storage(
+            cluster.config.volume_mounts, VOLUME_MOUNTS
```

Comment:
Can't we just do 
```
cluster.config.volume_mounts + VOLUME_MOUNTS
```
`generate_custom_storage` just looks like list addition from what I can see.